modules/claims_api/spec/sidekiq/poa_vbms_upload_job_spec.rb

```diff
@@ -259,6 +260,26 @@
           expect(power_of_attorney.vbms_error_message).to eq(e.message)
         end
       end
+
+      context 'when the BD upload refactor feature flag is enabled' do
+        before do
+          allow(Flipper).to receive(:enabled?).with(:claims_api_poa_uploads_bd_refactor).and_return true
+        end
+
+        it 'calls the PoaDocumentService before calling BD' do
```

Comment:
should this description just say 'calls the PoaDocumentService' since we aren't testing the call to BD?